modules/library/referencing/src/main/java/org/geotools/referencing/crs/DefaultProjectedCRS.java

```
@@ -299,4 +303,60 @@ protected String formatWKT(final Formatter formatter) {
         formatter.setLinearUnit(linearUnit);
         return "PROJCS";
     }
+
+    @Override
+    public String formatPROJ(PROJFormatter formatter) {
+        final Ellipsoid ellipsoid = ((GeodeticDatum) datum).getEllipsoid();
+        @SuppressWarnings("unchecked") // Formatter.setLinearUnit(...) will do the check for us.
+        final Unit<Length> unit = (Unit) getUnit();
+        final Unit<Length> linearUnit = formatter.getLinearUnit();
+        final Unit<Angle> angularUnit = formatter.getAngularUnit();
+        final Unit<Length> axisUnit = ellipsoid.getAxisUnit();
+        formatter.setLinearUnit(unit);
+        formatter.setAngularUnit(
+                DefaultGeographicCRS.getAngularUnit(baseCRS.getCoordinateSystem()));
+        OperationMethod method = conversionFromBase.getMethod();
+        if (method instanceof PROJFormattable) {
+            formatter.append((PROJFormattable) method);
+        }
```

What would happen if it's not a formattable? Do we need to throw an exception?